bindata/network/network-metrics/000-rbac.yaml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
minor nit: If this is part of Multus, then shouldn't it just be in `bindata/network/multus`? It can't be installed separately, right?

(It's in the `openshift-multus` namespace, so the answer to that question is presently no)

Reply:
It can't, as much as `multus-admission-controller` can't. I tried to stick to the same convention, see https://github.com/openshift/cluster-network-operator/tree/master/bindata/network/multus-admission-controller

It also look more clear what manifests belong to which component.
But if you prefer I can put the manifests under multus.